File: Android~/helium-android-bridge/src/main/java/com/chartboost/heliumsdk/unity/IBannerEventListener.java
package com.chartboost.heliumsdk.unity;

/**
 * Interface for sending real-time Banner events
 */
public interface IBannerEventListener {
    void DidLoadBanner(String placementName, int errorCode, String errorDescription);
    void DidShowBanner(String placementName, int errorCode, String errorDescription);
    void DidClickBanner(String placementName, int errorCode, String errorDescription);
    void DidWinBidBanner(String placementName, String auctionId, String partnerId, double price);
}

File: com.chartboost.mediation/Android~/chartboost-mediation-android-bridge/src/main/java/com/chartboost/mediation/unity/IBannerEventListener.kt
package com.chartboost.mediation.unity

/**
 * Interface for sending real-time Banner events
 */
@Deprecated("IBannerEventListener has been deprecated, utilize the new Banner API instead")
interface IBannerEventListener {
    fun DidLoadBanner(placementName: String, loadId: String, auctionId: String, partnerId: String, price: Double, lineItemName: String, lineItemId: String, error: String)
    fun DidClickBanner(placementName: String)
    fun DidRecordImpression(placementName: String)
}
